tools/dump-dataset.py

```diff
@@ -160,8 +161,7 @@ def dump_dataset(options):
         elif options.type == "stdout":
           extra = ""
           if target in dataset.labels and len(dataset.labels[target]) > 1:
-            assert dataset.can_serialize_data(target)
-            extra += " (%r)" % dataset.serialize_data(key=target, data=targets)
+            extra += " (%r)" % get_seq_labels_from_labels(targets, labels=dataset.labels[target])
```

Comment:
This change is not needed anymore. (It doesn't really make sense to adapt code to replace one deprecated function by another deprecated function.)

Also, you miss the `can_serialize_data` check here.